### ISSUE TITLE
tss2_provision fails if EK cert includes no crl dist point

### ISSUE THREAD

**Issue by rvka** [tpm2-software/tpm2-tss#1894]
On my Lenovo ThinkPad X1C6 with STM TPM, `tss2_provision` fails with
```
ERROR:fapi:src/tss2-fapi/ifapi_helpers.c:2576:ifapi_get_curl_buffer() curl_easy_perform() failed: URL using bad/illegal format or missing URL
ERROR:fapi:src/tss2-fapi/fapi_crypto.c:1645:get_crl_from_cert() ErrorCode (0x00060025) Get crl.
ERROR:fapi:src/tss2-fapi/fapi_crypto.c:1962:ifapi_verify_ek_cert() Get crl for intermediate certificate. ErrorCode (0x00060025)
ERROR:fapi:src/tss2-fapi/api/Fapi_Provision.c:813:Fapi_Provision_Finish() ErrorCode (0x00060025) Verify EK certificate
ERROR:esys:src/tss2-esys/esys_iutil.c:389:iesys_handle_to_tpm_handle() Error: Esys invalid ESAPI handle (ff).
ERROR:esys:src/tss2-esys/esys_iutil.c:1104:esys_GetResourceObject() Unknown ESYS handle. ErrorCode (0x0007000b)
ERROR:esys:src/tss2-esys/api/Esys_FlushContext.c:138:Esys_FlushContext_Async() flushHandle unknown. ErrorCode (0x0007000b)
ERROR:esys:src/tss2-esys/api/Esys_FlushContext.c:66:Esys_FlushContext() Error in async function ErrorCode (0x0007000b)
ERROR:fapi:src/tss2-fapi/fapi_util.c:1079:ifapi_session_clean() Cleanup session failed.
ERROR:fapi:src/tss2-fapi/api/Fapi_Provision.c:168:Fapi_Provision() ErrorCode (0x00060025) Provision
Fapi_Provision(0x60025) - fapi:No certificate
```

IIUC the problem is that the library fetches the EK certificate from the vendor URL (http://secure.globalsign.com/stmtpmekint05.crt), but then fails to fetch its CRL, because the certificate contains no CRL distribution points.  However, this is legitimate so the library should rather ignore this and proceed with the certificate.

**Comment by rvka** [comment]
Submitted https://github.com/tpm2-software/tpm2-tss/pull/1895

**Comment by AndreasFuchsSIT** [comment]
Fixed via #1895
